libsolidity/interface/StandardCompiler.cpp

```diff
@@ -655,7 +655,7 @@ std::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompiler:
 			string content = sources[sourceName]["content"].asString();
 			if (!hash.empty() && !hashMatchesContent(hash, content))
 				ret.errors.append(formatError(
-					false,
+					Error::Type::GeneralError,
 					"IOError",
 					"general",
```

Comment:
Also we could think about introducing a new category, such as `InterfaceError` or `CompilerError` in the standard json, which covers the "compiler interface" issues. Just like we have it in the exception system.

Reply:
Yea I think Chris suggested that too somewhere.